app/src/main/java/ghcharactertracker/com/ghcharactertracker/CharClass.java

```diff
@@ -0,0 +1,67 @@
+package ghcharactertracker.com.ghcharactertracker;
+
+/**
+ * Created by jesse.mailhot on 3/6/2018.
+ */
+
+public class CharClass {
+    ClassName name;
```

Comment:
Maybe make these instance variables private since you have getters and setters?

^^^ Apply this comment to all applicable files.